src/cycax/cycad/cycad_side.py

```diff
@@ -141,96 +208,23 @@ def slot(
             external_subtract=external_subtract,
         )
 
-    def _depth_check(self, val: float):
-        """
-        This is used to check the depth of an object. This method is just used for error catching.
-
-        Args:
-            val: Depth of object.
-
-        Raises:
-            ValueError: The side specidfied does not have a depth check method.
-        """
-        raise ValueError("_depth_check is Not implimented on" + self.name)
-
-    def _rotate(self):
-        """Rotate the part by 90 degress by keeping selected side on the same plain."""
-        msg = "_rotate should be defined for each side."
-        raise NotImplementedError(msg)
-
-    def rotate(self, angle: float = 90):
-        """Rotate the part by keeping the selected side on the same plain.
-
-        Rotation is counter clock wise. Negative angles are converted to poistive.
-
-        Args:
-            angle: The angle in degress that the part will be rotates on.
-
-        """
-        if self._parent.assembly is None:
-            # TODO: Think about which exception should be raised, maybe even a custom one.
-            msg = "Not part of an assembly"
-            raise ValueError(msg)
-
-        if angle % 90 != 0:
-            # TODO: Think about which exception should be raised, maybe even a custom one.
-            msg = "Can only rotate in multiples of 90 degrees"
-            raise ValueError(msg)
-
-        while angle < 0:
-            # Change the angle to a positive number.
-            angle = 360 + angle
-
-        for _n in range(int(angle / 90)):
-            self._rotate()
-
 
 class LeftSide(CycadSide):
-    name = "LEFT"
+    name = LEFT
```

Comment:
Avoid any weird string issues. Use the variable that define the side name not a new string